src/packages/v4/views/V4ProjectDashboard/ProjectHeaderStats.tsx

```diff
@@ -0,0 +1,68 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { ArrowTrendingUpIcon } from '@heroicons/react/24/outline'
+import { t, Trans } from '@lingui/macro'
+import ETHAmount from 'components/currency/ETHAmount'
+import { ProjectHeaderStat } from 'components/Project/ProjectHeader/ProjectHeaderStat'
+import { TRENDING_WINDOW_DAYS } from 'components/Projects/RankingExplanation'
+import { PropsWithChildren, useCallback } from 'react'
+import { twMerge } from 'tailwind-merge'
+import { useProjectPageQueries } from './hooks/useProjectPageQueries'
+import { useV4ProjectHeader } from './hooks/useV4ProjectHeader'
+
+export function ProjectHeaderStats() {
+  const { payments, totalVolume, last7DaysPercent } = useV4ProjectHeader()
+  const { setProjectPageTab } = useProjectPageQueries()
+
+  const openActivityTab = useCallback(
+    () => setProjectPageTab('activity'),
+    [setProjectPageTab],
+  )
+
+  return (
+    <div className="flex min-w-0 gap-12 md:shrink-0">
+      <a role="button" onClick={openActivityTab}>
+        <ProjectHeaderStat label={t`Payments`} stat={payments ?? 0} />
+      </a>
+      <ProjectHeaderStat
+        label={t`Total raised`}
+        stat={
+          <ETHAmount amount={BigNumber.from(totalVolume ?? 0)} precision={2} />
```

Comment:
mmm 🤔 . Shall we extend ETHAmount to accept bigints?

Reply:
Can you just slap a TODO on this in the code? I'll do my suggestion later if you like

Reply:
Yep we should, do you think extend or just switch it out for bigint's entirely? Added a TODO here now